### ISSUE TITLE
Support getting capabilities from "capabilities" key

### ISSUE THREAD

**Issue by vania-pooh** [aerokube/selenoid#384]
Currently can only work with "desiredCapabilities".

**Comment by vania-pooh** [comment]
E.g.
```
POST http://localhost:4444/wd/hub/session

{"capabilities":{"alwaysMatch":{"browserName":"firefox"}}}
```

**Comment by vania-pooh** [comment]
Fixed in 3225edd146347cf93dca9a031581ba59f410a273.
